htsget.md

```diff
@@ -162,6 +162,13 @@ The server SHOULD reply with an `UnsupportedFormat` error if the requested forma
 [^a]
 </td></tr>
 <tr markdown="block"><td>
+`class`  
+_optional string_
+</td><td>
+A list of URL classes to include, see below.
```

Comment:
I'd add the briefest explanation here too such as "allowing client to request the data header only, or to opt-out of receiving the header in subsequent requests"